resources/frontend_client/index_template.html

```diff
@@ -16,23 +16,34 @@
     <meta name="msapplication-config" content="/app/assets/img/browserconfig.xml">
     <meta name="theme-color" content="#ffffff">
     <meta name="apple-mobile-web-app-status-bar-style" content="default" />
+    <meta name="base-href" content="{{{baseHref}}}" />
```

Comment:
I realized we can switch from using triple braces to double braces in Mustache, which automatically escape HTML entities:

> All variables are HTML escaped by default. If you want to return unescaped HTML, use the triple mustache: {{{name}}}.

https://mustache.github.io/mustache.5.html

Then we'd have to remove `h.util/escape-html` for those variables.

But maybe it's better to leave it escaping in Clojure, before it even hits the Mustache template, in case someone accidentally changes to `{{{ }}}` for some reason.

Reply:
we were using triple braces before and AFAIK it wasn't escaping HTML entities. I'll have to double-check